File: src/main/resources/output/IFC4X3_csharp/Entities/IfcLightDistributionData/IfcLightDistributionData.cs
public class IfcLightDistributionData
{
    public IfcPlaneAngleMeasure MainPlaneAngle { get; set; }
    public List<IfcPlaneAngleMeasure> SecondaryPlaneAngle { get; set; }
    public List<IfcLuminousIntensityDistributionMeasure> LuminousIntensity { get; set; }
}

File: src/main/resources/output/IFC4X3_python/Entities/IfcLightDistributionData/IfcLightDistributionData.py
from typing import List

class IfcLightDistributionData:
    def __init__(self):
        self.MainPlaneAngle: IfcPlaneAngleMeasure = None
        self.SecondaryPlaneAngle: List[IfcPlaneAngleMeasure] = None
        self.LuminousIntensity: List[IfcLuminousIntensityDistributionMeasure] = None
